travis-scripts/ffosxbuild.sh

```diff
@@ -51,8 +51,10 @@ echo "Bundling Python libraries..."
 otool -L $workdir/bin/fontforge
 
 pylib=$(otool -L $workdir/bin/fontforge | grep -i python | sed -e 's/ \(.*\)//')
-pycruft=$(dirname $pylib)/../../..
-cp -a $pycruft/Python.framework $outdir/Contents/Frameworks
+pycruft=$(realpath $(dirname $pylib)/../../..)
+echo "pycruft: $pycruft"
+mkdir -p $outdir/Contents/Frameworks
+cp -av $pycruft/Python.framework $outdir/Contents/Frameworks
```

Comment:
Not worth changing, but I find that this is clearer if the destination parent has a `/` at the end.